src/ui.rs

```diff
@@ -118,11 +121,14 @@ impl Ui {
             combo_box_selected_item: style[imgui::StyleColor::Header],
             combo_box_selected_item_hovered: style[imgui::StyleColor::HeaderHovered],
             combo_box_selected_item_active: style[imgui::StyleColor::HeaderActive],
-            log_message_info: [0.70, 0.70, 0.70, 1.0],
+            log_message_info: [0.3, 0.3, 0.3, 1.0],
             log_message_warn: [0.80, 0.80, 0.05, 1.0],
             log_message_error: [1.0, 0.15, 0.05, 1.0],
             header_error: [0.85, 0.15, 0.05, 0.4],
             header_error_hovered: [1.00, 0.15, 0.05, 0.4],
+            tooltip_text: [1.0, 1.0, 1.0, 1.0],
```

Comment:
No need for this to exist if you always set it to white for both themes. Instead can be  `const COLOR_TOOLPTIP_TEXT`, defined somewhere at the top of the file.

Reply:
I'm not sure this remains the same for both themes. Also I prefer keeping setting regarding one thing together - it's already difficult enough to look for color setting all over the file.

Reply:
Ok, not a biggie.